casper/src/main/scala/coop/rchain/casper/engine/LastFinalizedStateTupleSpaceRequester.scala

```diff
@@ -27,8 +28,9 @@ object LastFinalizedStateTupleSpaceRequester {
 
   type StatePartPath = Seq[(Blake2b256Hash, Option[Byte])]
 
-  // TODO: extract this as a parameter (3000 on local machine is showing best results)
-  val pageSize = 3000
+  // TODO: extract this as a parameter
+  // 20,000 records uses about 2G of direct buffer memory
```

Comment:
2G is on importing node right? Not on exporting?

Reply:
I've updated the comment to refer to importing side.